backend/src/main/java/kr/touroot/member/dto/response/MyLikeTravelogueResponse.java

```diff
@@ -0,0 +1,30 @@
+package kr.touroot.member.dto.response;
+
+import java.time.format.DateTimeFormatter;
+import kr.touroot.travelogue.domain.Travelogue;
+import lombok.Builder;
+
+@Builder
+public record MyLikeTravelogueResponse(
+        long id,
+        String title,
+        String thumbnailUrl,
+        String createdAt,
+        String authorName,
+        String authorProfileImageUrl) {
```

Comment:
```suggestion
        String authorProfileImageUrl
) {
```

다른 레코드들과 컨벤션 맞추는 게 좋을 것 같습니다!

Reply:
와우 꼼꼼한 리뷰 감사합니다 :)